pkg/api/component_report.go

```diff
@@ -1188,18 +1194,27 @@ func (c *componentReportGenerator) assessComponentStatus(sampleTotal, sampleSucc
 			if c.MinimumFailure != 0 && (sampleTotal-sampleSuccess-sampleFlake) < c.MinimumFailure {
 				return apitype.NotSignificant, fischerExact
 			}
+
 			basisPassPercentage := float64(baseSuccess+baseFlake) / float64(baseTotal)
 			samplePassPercentage := float64(sampleSuccess+sampleFlake) / float64(sampleTotal)
+			effectivePityFactor := c.PityFactor
+			if approvedRegression != nil && approvedRegression.RegressedPassPercentage < int(basisPassPercentage) {
```

Comment:
Looks like basisPassPercentage is actually a ratio, needs to be `* 100`.  Same below on line 1204.  Once that is fixed, the squares get greened